samples/test-app/src/androidTest/kotlin/com/avito/android/ui/test/TouchOutsideActionTest.kt

```diff
@@ -0,0 +1,22 @@
+package com.avito.android.ui.test
+
+import com.avito.android.test.app.core.screenRule
+import com.avito.android.test.espresso.action.TouchOutsideAction
+import com.avito.android.ui.DialogActivity
+import junit.framework.TestCase.assertTrue
+import org.junit.Rule
+import org.junit.Test
+
+class TouchOutsideActionTest {
+
+    @get:Rule
+    val rule = screenRule<DialogActivity>(launchActivity = true)
+
+
+    @Test
+    fun outsideClicked_activityFinished() {
+        val activity = rule.activity
+        Screen.dialogScreen.root.interactionContext.perform(TouchOutsideAction())
```

Comment:
@dsvoronin What is a conventional way to `perform` actions?

Reply:
it is to add a function like `click()` or whatever on fitting `Action` class; but it's ok to just let it be like this; only real downside is discoverability for users